src/devices/arad_ms_meter.c

```diff
@@ -0,0 +1,123 @@
+/** @file
+    Arad/Master Meter Dialog3G water utility meter.
+    Copyright (C) 2022 avicarmeli
```

Comment:
needs an empty line above and below as in other decoders

Reply:
got it